### ISSUE TITLE
Comment before function name causes bareword where operator expected error

### ISSUE THREAD

**Issue by jidanni** [perltidy/perltidy#63]
I had to remove the comment in
```
my $fragment = $parser-> #parse_html_string
parse_balanced_chunk
($I);
```
making it
```
my $fragment = $parser->
parse_balanced_chunk
($I);
```
else
```
<stdin>: Begin Error Output Stream
<stdin>: 
<stdin>: 2: parse_balanced_chunk
<stdin>:    ^
<stdin>: found bareword where operator expected
<stdin>: 
<stdin>: Missing ';' above?
```

perltidy v20200110

**Comment by shancock9** [comment]
This was fixed in commit c2030cf.

**Comment by shancock9** [comment]
Version 20210625 has been released with a fix for this problem.
